runtime/sema/publicaccount_type.go

```diff
@@ -113,7 +113,7 @@ var PublicAccountType = func() *CompositeType {
 			PublicAccountPathsType,
 			publicAccountTypePathsFieldDocString,
 		),
-		NewUnmeteredPublicConstantFieldMember(
+		NewUnmeteredPublicFunctionMember(
```

Comment:
Just noticed: Is there any reason we do not have `PublicAccount.forEachPrivate` and `PublicAccount.forEachStored`? Those two functions would be very useful in scripts, e.g. for a wallet that is introspecting an account.

Reply:
These functions don't need to exist for scripts since we support `getAuthAccount` now; users can just `getAuthAccount` and call the functions directly from there.

Reply:
I see, that makes sense 👍

Reply:
Only downside is that using `getAuthAccount` instead is not possible in a transaction. We can add the functions it later if needed though